frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -149,6 +149,8 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
               onEnter={() => {
                 setFindMoreOpen(true);
                 setFieldValue('surplusFilter', true);
+                setFieldValue('includeAllProperties', true);
+                !keycloak.hasClaim(Claims.ADMIN_PROPERTIES) && setFieldValue('agencies', undefined);
```

Comment:
This removes any current agency filter?

Reply:
Yup, when they close they can alter as they please, and I had to set it to undefined otherwise when the user opened the Surplus Filter it would correctly switch to includeAllProperties but still have their agency applied against it

Reply:
But left it so SRES can run queries against specific agencies